mdtools/check.py

```diff
@@ -119,7 +119,7 @@ def array(
         axis is None and
         amin is None and
         amax is None and
-        dtype is None):
+            dtype is None):
```

Comment:
AFAIK this is also done by black but this is also debatable. My intent with this PR is to get rid of as many trivial changes (mostly white spaces) as possible. If one wants to run black than most changes are actual code changes instead of removing white spaces. My hope is that the changes done by black are easier to follow if there isn't any noise produced by white-space changes.

When looking at all possibilities ([in the third box from the bottom here](https://www.python.org/dev/peps/pep-0008/#indentation)) I actually prefer the way it's done by autopep8/black :-P

Reply:
> When looking at all possibilities (in the third box from the bottom here) I actually prefer the way it's done by autopep8/black :-P

Well, I like the first possibility with no extra indentation more :D

But black does it in yet another way, see the fourth box [here](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#how-black-wraps-lines).

But again, I would be fine with all these possibilities as long as it is consistent everywhere.

Reply:
Ah, good to know the way black does things. If black changes these things anyway they are not so important to be fixed in this PR. But these overhangs are only in a few places.